### ISSUE TITLE
add build support for arm64 on macos (Apple silicon "m1")

### ISSUE THREAD

**Issue by 7468696e6b** [SabakiHQ/Sabaki#774]
package.json?

electron 11 added macos arm64 support https://www.electronjs.org/blog/apple-silicon

edit: do want to note that the release (april 2020) runs well on rosetta 2, but I'm hoping for native arm64/universal support!

**Comment by 7468696e6b** [comment]
tried this:
in terminal first ran: `export npm_config_arch=arm64` and `export npm_config_target_arch=arm64`
then modified package.json
`"artifactName": "sabaki-v${version}-mac-arm64.${ext}"`
`    "electron": "^11.2.1",`
`    "electron-builder": "^22.10.4",`
`    "dist:macos": "npm run bundle && electron-builder -m --arm64",`
then ran `npm install` followed by `npm run build`
an app is produced in ./dist/mac-arm64
however, the app is unable to run 
<img width="239" alt="image" src="https://user-images.githubusercontent.com/61638924/106524580-c3fb9a80-64b0-11eb-9aa6-027cadba62bd.png">
confirmation of architecture 
<img width="257" alt="image" src="https://user-images.githubusercontent.com/61638924/106524648-dd044b80-64b0-11eb-8cef-315c5bfe3222.png">

**Comment by 7468696e6b** [comment]
the min version for arm support is 11 for electron
https://github.com/Webreaper/Damselfly/issues/88 for electron-builder

**Comment by 7468696e6b** [comment]
error data:
```log
Crashed Thread:        0  CrBrowserMain  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000010
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Segmentation fault: 11
Termination Reason:    Namespace SIGNAL, Code 0xb
Terminating Process:   exc handler [598]

VM Regions Near 0x10:
--> 
    __TEXT                      102f7c000-102fa4000    [  160K] r-x/r-x SM=COW  /Users/*/Documents/*/Sabaki.app/Contents/MacOS/Sabaki

Thread 0 Crashed:: CrBrowserMain  Dispatch queue: com.apple.main-thread
0   com.github.Electron.framework 	0x00000001050b5280 v8::internal::SetupIsolateDelegate::SetupHeap(v8::internal::Heap*) + 17904796

```

**Comment by 7468696e6b** [comment]
fixed in latest release 0.52.0 https://github.com/SabakiHQ/Sabaki/releases/download/v0.52.0/sabaki-v0.52.0-mac-arm64.7z https://github.com/SabakiHQ/Sabaki/releases (don't install from brew yet but install manually and requires self-sign or `sudo xattr -rd com.apple.quarantine Sabaki.app`)
